contributions/catalog/catalog_rest_service.py

```diff
@@ -50,6 +51,9 @@
             template_folder=template_dir)
 app.config.from_object(cfg)
 
+# App is behind one proxy that sets the -For and -Host headers.
```

Comment:
```suggestion
# TODO: Experimental fix for running behind the load balancer. Revisit to verify that this fix matches the actual deployment environment. Ref: https://werkzeug.palletsprojects.com/en/2.1.x/middleware/proxy_fix/
```
I'm suggesting a comment to note that we might need to verify this later. Thanks.